ChineseFilter/sections/specific.txt

```diff
@@ -74,4 +75,4 @@ nfmovies.com##a[href][onclick^="openurl()"] img[src^="/static/"]
 !+ PLATFORM(ios, ext_safari, ext_android_cb)
 nfmovies.com##body div[id^="aaaDiv"]
 !+ PLATFORM(windows, mac, android)
-||nfmovies.com/static/side.jpg$image,redirect=nooptext,domain=nfmovies.com
\ No newline at end of file
+||nfmovies.com/static/side.jpg$image,redirect=nooptext,domain=nfmovies.com
```

Comment:
Not seeing that picture in code.

Reply:
I also don't have this picture on my end.